full/full_coupler_mod.F90

```diff
@@ -1235,8 +1237,9 @@ end subroutine get_components_obj
 
   !> This subroutine finalizes the run
   subroutine coupler_end(Atm, Land, Ice, Ocean, Ocean_state, Land_ice_atmos_boundary, Atmos_ice_boundary,&
```

Comment:
I think the documentation for this routine should be updated for the added arguments/functionality

Reply:
Done!  @thomas-robinson, thought you might appreciate this.

Reply:
@mlee03 I was channelling my inner tom lol